frontend/src/Activity/History/Details/HistoryDetails.js

```diff
@@ -8,6 +8,33 @@ import DescriptionListItem from 'Components/DescriptionList/DescriptionListItem'
 import DescriptionListItemTitle from 'Components/DescriptionList/DescriptionListItemTitle';
 import DescriptionListItemDescription from 'Components/DescriptionList/DescriptionListItemDescription';
 
+function getDetailedPopoverBody(statusMessages) {
```

Comment:
Lets rename this function since in this instance its not going in a Popover components.